### PR TITLE
Move analytics to "ANALYTICS" environment variable

### PR DESCRIPTION
**Pull request by BenJam**
Generate analytics tracking if there’s an ENV variable called
ANALYTICS.

TODO
- [ ] set `ANALYTICS` env variable for GA (@andrew)

### CODE REVIEW

**Comment by andrew** [comment]
👍

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -29,16 +29,18 @@
   <%= content_for(:meta).presence || render_meta %>
   <link href="/opensearch.xml" title="Libraries.io" rel="search" type="application/opensearchdescription+xml">
   <% if Rails.env.production? %>
-    <script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
+      <script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
+  <% end %>
+  <% if ENV["ANALYTICS"] %>
```
Comment (andrew) [inline app/views/layouts/application.html.erb]: And you add a `.present?` on the end here, incase ANALYTICS is an empty string?

**Comment by andrew** [comment]
Will merge and deploy this tomorrow after I've set `ANALYTICS` on all the production servers

**Comment by andrew** [comment]
FYI I renamed the var to `GA_ANALYTICS_ID` in this commit: https://github.com/librariesio/libraries.io/commit/cde00f5257adb18acc06b0bbfcad68f3a523908d

**Comment by BenJam** [comment]
I'll get GA set up now then :)

**Pull request merged**